test/cloudtest/pkg/providers/packet/packet_provider.go

```diff
@@ -656,18 +656,18 @@ func (p *packetProvider) ValidateConfig(config *config.ClusterProviderConfig) er
 	for _, envVar := range config.EnvCheck {
 		envValue := os.Getenv(envVar)
 		if envValue == "" {
-			return errors.Errorf("environment variable are not specified %s Required variables: %v", envValue, config.EnvCheck)
+			return errors.Errorf("environment variable not set: %s. Required variables: %v", envVar, config.EnvCheck)
 		}
 	}
 
 	envValue := os.Getenv("PACKET_AUTH_TOKEN")
 	if envValue == "" {
-		return errors.New("environment variable are not specified PACKET_AUTH_TOKEN")
+		return errors.New("environment variable not set: PACKET_AUTH_TOKEN")
 	}
 
 	envValue = os.Getenv("PACKET_PROJECT_ID")
 	if envValue == "" {
-		return errors.New("environment variable are not specified PACKET_AUTH_TOKEN")
+		return errors.New("environment variable not set: PACKET_AUTH_TOKEN")
```

Comment:
should be PACKET_PROJECT_ID

Reply:
Actually, the code may be safely removed, because PACKET_PROJECT_ID and PACKET_AUTH_TOKEN are both listed in env-check: in packet.yaml

Reply:
*quickly looked on the env_check*.
 Yes, I think we can remove these checks.